assets/scss/_custom.scss

```diff
@@ -680,19 +680,6 @@ main.content {
   }
 }
 
-/* COMMUNITY legacy styles */
-/* Leave these in place until localizations are caught up */
-
-.newcommunitywrapper {
-  .news {
-    margin-left: 0;
-
-    @media screen and (min-width: 768px) {
-      margin-left: 10%;
-    }
-  }
-}
-
```

Comment:
Removed because it will interfere with the later on styling tweaks and improvements on #43740.

Reply:
Does this break any localization (did we check?)

Reply:
No it doesn't.

Reply:
No. I've checked all the pages.